ewasm-testnet-aleth-config.json

```diff
@@ -21,26 +22,25 @@
     "allowFutureBlocks" : false 
   }, 
   "genesis": { 
-    "nonce": "0x0000000000000000", 
-    "difficulty": "0x1", 
+    "nonce": "0x00", 
+    "difficulty": "0x01", 
     "mixHash": "0x0000000000000000000000000000000000000000000000000000000000000000", 
     "author": "0x0000000000000000000000000000000000000000", 
     "timestamp": "0x5b79de34", 
     "parentHash": "0x0000000000000000000000000000000000000000000000000000000000000000", 
     "extraData": "0x0000000000000000000000000000000000000000000000000000000000000000", 
     "gasLimit": "0x47b760" 
   }, 
-  "accounts": { 
+  "accounts": {
     "0000000000000000000000000000000000000001": { "precompiled": { "name": "ecrecover", "linear": { "base": 3000, "word": 0 } } },
     "0000000000000000000000000000000000000002": { "precompiled": { "name": "sha256", "linear": { "base": 60, "word": 12 } } },
     "0000000000000000000000000000000000000003": { "precompiled": { "name": "ripemd160", "linear": { "base": 600, "word": 120 } } },
     "0000000000000000000000000000000000000004": { "precompiled": { "name": "identity", "linear": { "base": 15, "word": 3 } } },
-    "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp", "startingBlock" : "0x2dc6c0" } },
-    "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "startingBlock" : "0x2dc6c0", "linear": { "base": 500, "word": 0 } } }, 
-    "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "startingBlock" : "0x2dc6c0", "linear": { "base": 40000, "word": 0 } } }, 
-    "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product", "startingBlock" : "0x2dc6c0" } },
-    "000000000000000000000000000000000000000a": { "codeFromFile": "dummymetering.wasm", "balance": "0x01", "nonce": "0x00", "storage": {} },
-    "000000000000000000000000000000000000000b": { "codeFromFile": "dummyevm2wasm.wasm", "balance": "0x01", "nonce": "0x00", "storage": {} },
-    "9fa4f23079bae7a7a5c392b67e18093310315bfc": { "balance": "100000000000000000000000000", "storage": {}, "nonce": "0x0", "code": "" }
-  } 
+    "0000000000000000000000000000000000000005": { "precompiled": { "name": "modexp" } },
+    "0000000000000000000000000000000000000006": { "precompiled": { "name": "alt_bn128_G1_add", "linear": { "base": 500, "word": 0 } } }, 
+    "0000000000000000000000000000000000000007": { "precompiled": { "name": "alt_bn128_G1_mul", "linear": { "base": 40000, "word": 0 } } }, 
+    "0000000000000000000000000000000000000008": { "precompiled": { "name": "alt_bn128_pairing_product" } },
+    "000000000000000000000000000000000000000a": { "codeFromFile": "metering.wasm", "balance": "0x01", "nonce": "0x00", "storage": {} },
+    "9fa4f23079bae7a7a5c392b67e18093310315bfc": { "balance": "0x200000000000000000000000000000000000000000000000000000000000000", "storage": {}, "nonce": "0x0", "code": "" }
```

Comment:
Balance in geth seems to be:
```
    "9fa4f23079bae7a7a5c392b67e18093310315bfc": {
      "balance": "100000000000000000000000000"
    }
```

Reply:
Something else must have changed with the genesis we have in staging because I am getting genesis mismatch when trying to connect after making this change.

Reply:
Okay it syncs against staging.  We need a tagged release so that https://github.com/ewasm/testnet/pull/143 can be merged and then, the aleth branches changed here and this PR merged.

Reply:
What do you mean exactly?